protoc-gen-twirp_ruby/main.go

```diff
@@ -189,24 +224,23 @@ func writeGenResponse(w io.Writer, resp *plugin.CodeGeneratorResponse) {
 // e.g. toRubyType(".foo.my_message", []string{}) => "Foo::MyMessage"
 // e.g. toRubyType(".foo.my_message", []string{"Foo"}) => "MyMessage"
 // e.g. toRubyType("google.protobuf.Empty", []string{"Foo"}) => "Google::Protobuf::Empty"
-func toRubyType(protoType string, currentModules []string) string {
-	rubyConsts := splitRubyConstants(protoType)
-	if len(rubyConsts) == 0 {
-		return ""
+func (g *generator) toRubyType(protoType string) string {
```

Comment:
This change requires some small refactoring in the `TestToRubyType` test in https://github.com/twitchtv/twirp-ruby/blob/master/protoc-gen-twirp_ruby/main_test.go#L38-L60.